website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,162 @@
+will not be able to use your operator. Therefore, your operator should **always apply at least resource requests for CPU and
+memory** for `Pods/Deployments` that it creates as part of reconciliation. Ideally your operator also applies resource 
+limits to those `Pods/Deployments`. 
+
+#### General guidelines
+
+**Note that pretty much all controllers consume:**
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+Operator authors cannot anticipate how they scale in usage in all customer environments, so setting one-size-fits-all 
+that fits well for all scenarios is not possible then, clusters administrators must be allowed to change the values 
+in order to respond to changes in their clusters’ needs, such as by resizing control plane nodes to provide more resources.
+
+**Following are some general recommendations to manage the resources:**
+
+- **MUST** declare resource requests for both CPU and memory.
+- **OPTIONALLY** setting the resources limit for Memory and CPU (Note that if your project is integrated with OLM then, admins would be able to customize it via the [subscription][olm-subscriptions]). Also, be aware that our project was built with Kubebuilder or SDK scaffold then for the `Operator/manager`(See `config/manager/manager.yaml`), then some values which allow you to get started are scaffolded by default, however, you ought to optimize based on the needs.
+- **Should** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
+- **OPTIONALLY** ensuring that the Operators uses [liveness health check][k8s-probes] which can be used to check the `CPU` performance. If your project was built with Kubebuilder or SDK scaffold then, it is scaffolded by default for the `Operator/manager`. (See `config/manager/manager.yaml`).
+- **Should** allow admins to customize the `requests/limits` amounts defined for the Pod/Deployment Operands created by the Operator and not keep them fixed on the code.
+- **Should** document how cluster admins can customize/rightsize the resources limits for the Operator and Operands.
+
+You also might consider providing solutions that allow vertical pod autoscaler to manage arbitrary scaling for your Operators. [More info][k8s-vertical-pod-scaling]. 
+And then, you also might look for allowing [horizontal pod autoscaling][k8s-autoscaling] for the Operands. In this case, see that resource requests are also needed for horizontal pod autoscaling to work.
```

Comment:
Nitpicking: you might also look at allowing... In this case, be aware that resource requests are also required for horizontal pod autoscaling to work